src/app/workflow/launch-third-party/launch-third-party.component.ts

```diff
@@ -174,6 +168,11 @@ export class LaunchThirdPartyComponent extends Base implements OnChanges, OnInit
     iconRegistry.addSvgIcon('dnanexus', sanitizer.bypassSecurityTrustResourceUrl('../assets/images/thirdparty/DX_Logo_white_alpha.svg'));
     iconRegistry.addSvgIcon('terra', sanitizer.bypassSecurityTrustResourceUrl('../assets/images/thirdparty/terra.svg'));
     iconRegistry.addSvgIcon('anvil', sanitizer.bypassSecurityTrustResourceUrl('../assets/images/thirdparty/anvil.svg'));
+    iconRegistry.addSvgIcon('bdc_terra', sanitizer.bypassSecurityTrustResourceUrl('../assets/images/thirdparty/bdc_terra.svg'));
+    iconRegistry.addSvgIcon(
+      'bdc_seven_bridges',
+      sanitizer.bypassSecurityTrustResourceUrl('../assets/images/thirdparty/bdc_seven_bridges.svg')
```

Comment:
Does this need to be checked-in? Don't see it at https://github.com/dockstore/dockstore-ui2/tree/feature/3123/bdcatalyst/src/assets/images/thirdparty

Reply:
D'oh! I'd been experimenting with their artwork, turned out to be too big, and forgot to remove this.